universal-application-tool-0.0.1/app/assets/javascripts/azure_upload.ts

```diff
@@ -9,7 +9,10 @@ class AzureUploadController {
     const uploadContainer = document.getElementById(
       AzureUploadController.UPLOAD_CONTAINER_ID
     )
-    uploadContainer.addEventListener('submit', (event) =>
+
+    const uploadButton = document.getElementById('cf-block-submit')
```

Comment:
FYI this line might be what's breaking the dev file upload since there's no `cf-block-submit` element in the dev upload view (so the file isn't being submitted).

Reply:
Ahh thank you!!! Fixed